src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -711,9 +712,23 @@ export class PostgresCohortMembersService {
           HttpStatus.NOT_FOUND
         );
       }
-      let result = await this.cohortMembersRepository.save(
-        cohortMembershipToUpdate
-      );
+
+      // Iterate over all keys in cohortUpdateDto
+      for (const key in cohortMembersUpdateDto) {
+        if (
+          cohortMembersUpdateDto.hasOwnProperty(key) &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)